### DISCUSSION TITLE
Welche Seiten brauchen wir?

### DISCUSSION THREAD

**Discussion by Marc738** [SE-TINF22B2/G3-ApeRepublic#62] [Ideas]
https://miro.com/app/board/uXjVNNV8BIE=/
### Prio 1 Seiten
* Login-Seite
* Registieren-Seite
* Passwort vergessen-Seite (Reset-Passwort-Seite)
* Übersicht-Seite
* Account-Seite (Vom User selbst)
* Aktien-Seite

### Prio 2 Seiten
* Welcome-Seite
* User-Seite (Um andere User anzuschauen)
* Einstellungsseite

**Comment by MatejEvc** [comment]
Account-Seite = User-Seite?

**Reply by Marc738** [reply]
Ne, die User-Seite soll angezeigt werden wenn man nach einem anderen User gesucht hat und seine Performance und so anschauen will.
Account-Seite ist das was man wörtlich darunter versteht.

**Comment by elnaz-gharoon** [comment]
In Passwort-Vergessen Seite muss der Benutzer seine Email eingeben, um eine Bestätigung von Email zu erhalten
In Passwort-Reset Seite muss der Bentzer seine neues Passwort 2 Mal bestimmen 
Richtig ? @Marc738

**Reply by Marc738** [reply]
Ja, würde ich genauso machen. Sry wegen später antwort, habs erst jetzt gesehen :)

**Comment by elnaz-gharoon** [comment]
Unterschied zwische Aktien und Übersicht-Seite?

**Reply by MatejEvc** [reply]
Ich glaube die Übersicht-Seite soll eher dein Portfolio zeigen, und Aktien-Seite eine einzelne Aktie
